graphql_compiler/compiler/expressions.py

```diff
@@ -785,7 +799,18 @@ def to_cypher(self):
 
     def to_sql(self, aliases, current_alias):
         """Return a sqlalchemy Column picked from the appropriate alias."""
-        # get the type of the folded field
+        # _x_count is a special case that has already been coalesced to 0.
+        # _x_count's intermediate output name is always fold_output__x_count
```

Comment:
Maybe document this expectation in the definition of `CompilationState._aliases`. I think that's the easiest way to make sure:
- it actually gets implemented (or the implementation does not get removed)
- the usage of it is not confusing (it is backed by the spec of the field)